adafruit_ht16k33/segments.py

```diff
@@ -26,6 +26,7 @@
 """
 
 from time import sleep
+#from adafruit_ht16k33.ht16k33 import HT16K33
```

Comment:
Just a minor nitpick, can we remove this commented out line please?

Reply:
Done! I would have nitpicked that too.